### PR TITLE
added core contributors section with contributors and criteria

### PR DESCRIPTION
**Pull request by geekygirldawn**
As discussed in issue 126. Note that this should not close that issue, since this pull request only addresses part of that issue.

### CODE REVIEW

**Comment by GeorgLink** [comment]
Thanks for advancing this @geekygirldawn.
I did some digging to find additional contributors and added a more complete list.

File: README.md [hunk 1]
```diff
@@ -91,15 +91,52 @@ Most of the work in the metrics side will be done through pull requests to the m
...
+
+If you'd like to be on our squad, an easy way to start is by going through the 
+[issue list](https://github.com/chaoss/wg-diversity-inclusion/issues) and fixing some. :tada:
+
+**All Contributors**
+Ordered by first name
+
+[Akshita Gupta](https://github.com/akshitac8), 
+Alexander Serebrenik, 
+Brian Proffitt,
```
Comment (dicortazar) [inline README.md]: https://github.com/bproffitt

File: README.md [hunk 2]
```diff
@@ -91,15 +91,52 @@ Most of the work in the metrics side will be done through pull requests to the m
...
+People not participating over a 3 month period may be removed as core contributors.
+
+If you'd like to be on our squad, an easy way to start is by going through the 
+[issue list](https://github.com/chaoss/wg-diversity-inclusion/issues) and fixing some. :tada:
+
+**All Contributors**
+Ordered by first name
+
+[Akshita Gupta](https://github.com/akshitac8), 
+Alexander Serebrenik,
```
Comment (dicortazar) [inline README.md]: https://github.com/aserebrenik

**Comment by geekygirldawn** [comment]
+1 Great additions, thank you!

**Comment by GeorgLink** [comment]
Thanks @dicortazar , I added the missing profile links

**Comment by GeorgLink** [comment]
Thanks @geekygirldawn and @dicortazar

**Pull request merged**